src/exp.jl

```diff
@@ -26,48 +26,49 @@ const min_exp2(::Type{Float32}) = -150f0
     c3  = 0.5550410866482046596e-1
     c2  = 0.2402265069591012214
     c1  = 0.6931471805599452862
-    return @horner x c1 c2 c3 c4 c5 c6 c7 c8 c9 c10 c11
+    @horner x c1 c2 c3 c4 c5 c6 c7 c8 c9 c10 c11
```

Comment:
can you please leave in the explicit return, thanks.

Reply:
Added explicit returns.